src/borg/testsuite/archiver.py

```diff
@@ -2725,6 +2726,27 @@ def test_extract_hardlinks(self):
             assert os.stat('input/dir1/aaaa').st_nlink == 2
             assert os.stat('input/dir1/source2').st_nlink == 2
 
+    def test_detect_attic_repo(self):
+        path = attic_repo(self.repository_path)
+        cmds = [
+            ['create', path + '::test', self.tmpdir],
+            ['extract', path + '::test', 'output.tar'],
```

Comment:
Good catch. Should be dropped. I will wait for further comments/reviews before submitting a fix, though...